src/main/java/homewrok_nr_14/DivisibleBy7.java

```diff
@@ -0,0 +1,28 @@
+package homewrok_nr_14;
+
+public class DivisibleBy7 {
+    public static void main(String[] args) {
+
+        countDivisibleBy7Numbers(1, 22334);
+
+
+    }
+
+    public static void countDivisibleBy7Numbers(int inferiorLimit, int superiorLimit) {
+        int countDivisors = 0;
+        for (int i = inferiorLimit; i <= superiorLimit; i++) {
+            if (i % 7 == 0) {
+                countDivisors++;
+            }
+        }
+                System.out.println("the number of divisible by 7 : " + countDivisors);
+
+        if (inferiorLimit > superiorLimit) {
+            try {
```

Comment:
this doesn't make sense. you're throwing an exception and catching it. this is because you cannot throw it because this exception is a checked one. Make the exceptions runtime exceptions and ditch the try catch